### PR TITLE
Changes to readme for installation steps

### PR DESCRIPTION
**Pull request by andram11**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Issue link

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature updates / changes
- [ ] Tests
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

I assume no testing needed for this kind of change

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have updated the change log

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,16 +52,22 @@ NODE_ENV=test
 
 ## Installation
 
-To install all the project's dependencies run:
+If you don't have yarn installed, make sure you have a package manager like npm installed first, and then install yarn:
```
Comment (Dan-Y-Ko) [inline README.md:55]: I'm not sure if it's a good idea to document this as the way to install yarn: https://yarnpkg.com/getting-started/install. 
https://classic.yarnpkg.com/lang/en/docs/install/#windows-stable

Comment (cherylli) [reply README.md:55]: yeah this is new for yarn 2+ I think I'm still using yarn 1, I don't think it makes a difference but we can definitely include this yarn install link.

**Review by cherylli — CHANGES_REQUESTED** [review]
Looks good, we could add that link Dan posted and also please also update the changelog. Thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,15 +17,17 @@
...
 
-- Added same site property to the clear cookies function ([#218](https://github.com/chingu-x/chingu-dashboard-be/pull/218))
-- Added routes for teams to create own tech stack categories([#208](https://github.com/chingu-x/chingu-dashboard-be/pull/208))
-- Added unit tests for Features controller and services ([#220](https://github.com/chingu-x/chingu-dashboard-be/pull/220))
-- Added GET endpoint for solo project ([#223](https://github.com/chingu-x/chingu-dashboard-be/pull/223))
+-   Added same site property to the clear cookies function ([#218](https://github.com/chingu-x/chingu-dashboard-be/pull/218))
+-   Added routes for teams to create own tech stack categories([#208](https://github.com/chingu-x/chingu-dashboard-be/pull/208))
+-   Added unit tests for Features controller and services ([#220](https://github.com/chingu-x/chingu-dashboard-be/pull/220))
+-   Added GET endpoint for solo project ([#223](https://github.com/chingu-x/chingu-dashboard-be/pull/223))
+-   Updated readme for installation part ([#225])(https://github.com/chingu-x/chingu-dashboard-be/pull/225)
```
Comment (cherylli) [inline CHANGELOG.md:24]: Should probably be in the `### Changed` section.

If you can please also try to revert all the formatting changes. Thanks!

Comment (andram11) [reply CHANGELOG.md]: I have moved it but I can't see the formatting changes in my ide, for me it looks normal....:(

Comment (cherylli) [reply CHANGELOG.md]: https://github.com/chingu-x/chingu-dashboard-be/pull/225/files

You can see in this page, looks like a space or two is added in each line

Comment (Dan-Y-Ko) [reply CHANGELOG.md]: @andram11 I had issues with this too. I don't really know how to fix it, but the easiest solution for me was to manually update the changelog directly from the repo

**Review by cherylli — APPROVED** [review]
Looks good. Thanks for the update!

File: CHANGELOG.md [hunk 3]
```diff
@@ -34,9 +34,11 @@
 - Added units test for sprints ([#224](https://github.com/chingu-x/chingu-dashboard-be/pull/224))
 
 ### Changed
+
 - Updated cors origin list ([#218](https://github.com/chingu-x/chingu-dashboard-be/pull/218))
 - refactored unit tests for the ideations controller and services([#219](https://github.com/chingu-x/chingu-dashboard-be/pull/219))
 - revised tech selections route to update only one tech per request([#221](https://github.com/chingu-x/chingu-dashboard-be/pull/221))
+- Updated readme for installation part ([#225])(https://github.com/chingu-x/chingu-dashboard-be/pull/225)
```
Comment (cherylli) [inline CHANGELOG.md:41]: THis one needs to be moved to the unrelease section now, after the latest release merge. 🙏

**Review by cherylli — CHANGES_REQUESTED** [review]

**Comment by cherylli** [comment]
Thanks for the update but it seems there are also a few other files changed with the new update. The formatting seems to match the prettier file which changed the tabWidth from 2-4, were these files not formatted by husky @timDeHof ? these look like your latest changes

**Comment by cherylli** [comment]
@JoshuaHinman are you able to review this please. If not we can see if anyone else is available to review this

**Review by JoshuaHinman — APPROVED** [review]
Look great!

**Pull request merged**